spec/factories/curator/metastreams/descriptives.rb

```diff
@@ -14,7 +14,6 @@
     related { create(:curator_descriptives_related) }
     title { create(:curator_descriptives_title_set) }
     subject_other { create(:curator_descriptives_subject) }
```

Comment:
Should this be replaced with `digital_origin { 'reformatted_digital' }` ? Or is there an implied default so it's no longer needed?

Reply:
It has that set as a default